concepts/05 Widgets/Popover/05 Customize the Appearance/05 Customize the Content/5 Switching Templates On the Fly.md

```diff
@@ -26,7 +26,7 @@ If you need to render different templates depending on a specific condition, def
         $("#buttonContainer").dxButton({
             text: "Change the Template", 
             onClick: function (e) {
-                var currentTemplate = popover.option("contentTemplate");
+                const currentTemplate = popover.option("contentTemplate");
```

Comment:
@ksercs The same here. The name currentTemplate shows thta it can be changed dynamically.

Reply:
the same here